openc3/lib/openc3/logs/log_writer.rb

```diff
@@ -287,8 +289,12 @@ def prepare_write(time_nsec_since_epoch, data_length, redis_topic, redis_offset)
           Logger.debug("Log writer start new file due to cycle size #{@cycle_size}")
           start_new_file()
         elsif @enforce_time_order and @previous_time_nsec_since_epoch and (@previous_time_nsec_since_epoch > time_nsec_since_epoch)
-          Logger.debug("Log writer start new file due to out of order time: #{Time.from_nsec_from_epoch(@previous_time_nsec_since_epoch)} #{Time.from_nsec_from_epoch(time_nsec_since_epoch)}")
-          start_new_file()
+          # Warning: Creating new files here can cause lots of files to be created if packets make it through out of order
+          # Changed to just a error to prevent file thrashing
+          unless @out_of_order
+            Logger.error("Log writer out of order time detected (increase buffer depth?): #{Time.from_nsec_from_epoch(@previous_time_nsec_since_epoch)} #{Time.from_nsec_from_epoch(time_nsec_since_epoch)}")
```

Comment:
Let's reference the exact variable: TLM_BUFFER_DEPTH
"Log writer out of order time detected! Create or increase TLM_BUFFER_DEPTH in plugin.txt."
Is there anyway we could give them the offending target name?

Reply:
I was just looking for something to search on. Perhaps you want to add this to the demo somewhere as an example? I noticed that we have a number of things not in the demo right now so if it doesn't get done in this PR we should take an action to add more keywords, features, etc to the demo.